.github/CODEOWNERS

```diff
@@ -1,12 +1,20 @@
 # Each line is a file pattern followed by one or more owners.
 # See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
+/src/data/datasets.js @DataBiosphere/data-explorer-eng
 /src/libs/brand*.js @DataBiosphere/terra-cobranding
 /src/libs/colors.js @DataBiosphere/terra-cobranding
 /src/pages/Environments.js @DataBiosphere/broad-interactive-analysis
 /src/libs/ajax/Apps.js @DataBiosphere/broad-interactive-analysis
 /src/libs/ajax/Runtimes.js @DataBiosphere/broad-interactive-analysis
 /src/libs/ajax/AzureStorage.js @DataBiosphere/broad-interactive-analysis
 /src/libs/ajax/GoogleStorage.js @DataBiosphere/broad-interactive-analysis
+/src/libs/ajax/Catalog.ts @DataBiosphere/data-explorer-eng
+/src/pages/library/DataBrowser* @DataBiosphere/data-explorer-eng
+/src/pages/library/dataBrowser-utils* @DataBiosphere/data-explorer-eng
+/src/pages/library/Datasets* @DataBiosphere/data-explorer-eng
```

Comment:
Maybe we should backlog a ticket to reorganize wildcard files into wildcard directories. Files seem bad since a user might not know about the rule when creating a file (e.g., `DataBrowserForWorkspaces.js`).

Reply:
I like that. I'll make a ticket.

Reply:
https://broadworkbench.atlassian.net/browse/DC-658